security-analysis/security-analysis-api/src/main/java/com/powsybl/security/interceptors/DefaultSecurityAnalysisInterceptor.java

```diff
@@ -18,17 +20,28 @@
 public class DefaultSecurityAnalysisInterceptor implements SecurityAnalysisInterceptor {
```

Comment:
Sonar require an explanation when a method is let empty. Please every method in this class